amicleanup/amicleanup.py

```diff
@@ -75,8 +76,9 @@ def get_orphaned_images(ec2, filters, retention):
                     if tag['Key'] == 'ops:retention':
                         retention = int(tag['Value'])
 
-            if retention and
-                    creation_date < (datetime.now() - timedelta(days=retention)):
+            if retention and \
+                    creation_date \
```

Comment:
Is this a PEP8 thing? I think it's kind of ugly to read, personally.

Reply:
Yea... I can revert this. I was just making sure `pycodestyle` passed since I had to make changes in the first place.

Reply:
I'm fine with leaving it, especially if it is required to get a passing grade from a linter we've adopted.